### ISSUE TITLE
Scatter Shapes and Legend Shapes

### ISSUE THREAD

**Issue by tgvoskuilen** [PhilJay/MPAndroidChart#3771]
I can set up a scatter chart with multiple shaped markers (square, circle, triangle, cross, X, chevron up, and chevron down) but the only legend form options are square and circle. How can I set a custom LegendEntry with these additional shapes?

**Comment by ankitgoyal1009** [comment]
I don't think that is possible, right now only circle and square is supported. You can use your own TextView/ImageView for this purpose.
